cclib/parser/molcasparser.py

```diff
@@ -217,31 +217,30 @@ def extract(self, inputfile, line):
         #  Threshold for Fock matrix                  0.15E-03
         if line[:34] == '++    Optimization specifications:':
 
+            target_array = []
```

Comment:
Rather than `target_array` and `value_array`, can you use `scftargets` and `scfvalues`? You use that terminology for `atomcoords`, and we use it throughout the other parsers to indicate a temporary list/array.

Reply:
Makes sense.